pkg/server/const.go

```diff
@@ -0,0 +1,19 @@
+package server
+
+const (
+	PassthroughHeader   = "X-Kcp-Api-V1-Discovery-Passthrough"
+	WorkspaceAnnotation = "tenancy.kcp.dev/workspace"
+)
+
+type (
+	acceptHeaderContextKeyType int
+	userAgentContextKeyType    int
+)
+
+const (
+	// clusterKey is the context key for the request namespace.
+	AcceptHeaderContextKey acceptHeaderContextKeyType = iota
```

Comment:
don't make these public please. They are no means for consumption. If we have to open them, we add `WithAcceptHeader`and `AcceptHeaderFrom`, without leaking the key.